File: src/db/sormas_db.R
library(DBI)

do_connect <- function(){
# Connect to postgres database
con <- dbConnect(
        RPostgres::Postgres(),
        dbname = 'sormas', 
        host = 'postgres',
        port = 5432,
        user = 'postgres',
        password = 'password'
)
# dbListTables(con)
return(con)
}

# mockup, write to DB later
do_writeback <- function(df,filename){
    f <- paste('/srv/output/',filename, sep="")
    saveRDS(df, file=f)
}

File: src/python/db/sormas_db.py
import psycopg2

def sormas_db_connect(host="postgres", port="5432", dbname="sormas", user="sormas_reader", password="password"):
    conn = psycopg2.connect(host=host, port=port, dbname=dbname, user=user, password=password)
    return conn
